submissions/Shramkoweb/html-css-popup/source/sass/blocks/main-nav.scss

```diff
@@ -0,0 +1,83 @@
+.main-nav {
+  position: relative;
+
+  padding: 2em 2em;
+}
+
+.main-nav__list {
+  @include list-reset;
+
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.main-nav__item {
+  position: relative;
+
+  margin-right: 10px;
+}
+
+.main-nav__checkbox {
+  &:checked ~ .pop-up {
+    display: block;
+  }
+}
+
+.main-nav__link {
+  &:hover {
+    text-decoration: underline;
+  }
+
+  &--user {
```

Comment:
Try to avoid write short selector. It's hard to find right selector during debugging code. Better write full selector: `.main-nav__link--user` instead of `.main-nav__link &--user`

Reply:
In VS Code when you move cursor to &--modificator thy compiled css selector to full name. 
But short selector save my time =)

Reply:
Thx. You are right.